esinstall/src/rollup-plugins/rollup-plugin-wrap-install-targets.ts

```diff
@@ -98,8 +132,8 @@ export function rollupPluginWrapInstallTargets(
           normalizedFileLoc.includes(`node_modules/${p}${p.endsWith('.js') ? '' : '/'}`),
         );
         const cjsExports = isExplicitAutoDetect
-          ? cjsAutoDetectExportsRuntime(val)
-          : cjsAutoDetectExportsStatic(val);
+          ? cjsAutoDetectExportsTrusted(val)
+          : cjsAutoDetectExportsUntrusted(val);
```

Comment:
chose more appropriate names, now that there's a (sandboxed) runtime aspect to our untrusted scanner.